packages/env/lib/init-config.js

```diff
@@ -176,3 +237,29 @@ RUN echo 'xdebug.mode=${ enableXdebug }' >> /usr/local/etc/php/php.ini
 RUN echo '${ clientDetectSettings }' >> /usr/local/etc/php/php.ini
 	`;
 }
+
+/**
+ * Generates the Dockerfile used by wp-env's `cli` and `tests-cli` instances.
+ *
+ * @param {string} image The base docker image to use.
+ *
+ * @return {string} The dockerfile contents.
+ */
+function cliDockerFileContents( image ) {
+	return `FROM ${ image }
+
+# Switch to root so we can create users.
+USER root
```

Comment:
It seems like it could be worth deduplicating some of this config, just because it's so similar and it could be easy for a change to not get copied to the other if needed

Reply:
Since these two files have to be distinct, there isn't much that we could duplicate, unfortunately.